src/Compilers/CSharp/Portable/CodeGen/EmitOperators.cs

```diff
@@ -506,6 +500,55 @@ private void EmitCondExpr(BoundExpression condition, bool sense)
             return;
         }
 
+        /// <summary>
+        /// Emits boolean expression without branching if possible (i.e., no logical operators, only comparisons).
+        /// Leaves a boolean (int32) value on the stack which conforms to sense, i.e., <c>condition == sense</c>.
+        /// </summary>
+        private bool TryEmitCompExpr(BoundExpression condition, bool sense)
+        {
+            RemoveNegation(ref condition, ref sense);
```

Comment:
>RemoveNegation(ref condition, ref sense); [](http://example.com/codeflow?start=12&length=41)

Is significance of this operation tested?

Reply:
Not enough, added another test for that.